dind/cmd/cluster-up/main.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"time"
+
+	"github.com/golang/glog"
+	"k8s.io/test-infra/dind/pkg/cluster"
+	"k8s.io/test-infra/dind/pkg/util"
+)
+
+const (
+	masterConfigDir = "/var/kubernetes"
+	kubecfg         = "/var/kubernetes/admin.conf"
+	timeout         = time.Duration(5) * time.Minute
+)
+
+var (
+	loadImage = true
+	nodeImage = "k8s.gcr.io/dind-node-amd64"
+	proxyAddr = ""
+	version   = ""
+	numNodes  = 4
+)
+
+func init() {
+	flag.BoolVar(&loadImage, "load-image", true, "If the image needs to be loaded from the file-system.")
+	flag.StringVar(&nodeImage, "node-image", nodeImage, "The node image to use.")
```

Comment:
Pull spec? "image" is vague?

Reply:
But it's not a pull spec, because the dind image might not be pulled. It's probably going to be side loaded into docker instead.

Maybe call it the "dind-node-image"?